django_saas_email/models.py

```diff
@@ -154,34 +192,39 @@ def create_mail(self, template_name, context, to_address, from_address=None, sub
         except ValidationError:
             raise ValueError("The given email is not valid: {}".format(to_address))
 
-        mail = self.create(template=template, context=context_json, from_address=from_address, to_address=to_address,
-                           subject=subject)
+        mail = self.create(
+            template=template,
+            context=context_json,
+            from_address=from_address,
+            to_address=to_address,
+            subject=subject,
+        )
+
+        if attachments is not None:
```

Comment:
Assuming you defined `preselected_attachments` as a field, here you should merge the `attachments` passed as an argument with any `preselected_attachments` of the chosen `template`.

Reply:
I thought that preselected attachments are mostly for frontend to display suggested templates. Not sure that merging is required. @jensneuhaus could you please clarify?